.travis.yml

```diff
@@ -14,20 +21,26 @@ branches:
   - master
   - dev
 before_install:
-  - yarn
+  # cordova currently only works with node 6.x
+  - curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+  - sudo apt-get install -y nodejs
```

Comment:
@brucebinary If you use `dist : trusty`, you don't need `sudo` (if I remember correctly)

Reply:
I remember that sudo not being needed for one of the distributions as well. I'll see if I can remove sudo to keep things simple abit

Reply:
@arnabk nope doesn't work.